graphs/DataVisualisation/index.js

```diff
@@ -66,21 +75,26 @@ module.exports = React.createClass({
   },
 
   renderDeltaArrow: function() {
-    var props   = this.props,
-        delta   = props.delta,
-        loading = props.loading;
+    var props      = this.props,
+        delta      = props.delta,
+        loading    = props.loading,
+        emptyState = props.emptyState;
 
-    if (loading || delta) {
+    if (emptyState) {
+      return <DeltaArrow delta={ delta } emptyState={ emptyState } />;
+    }
+    else if (loading || delta) {
```

Comment:
I think that if it doesn't meet any of the conditions emptyState, loading or delta then it is not supposed to render a DeltaArrow.